tests/wpunit/Tribe/Context_LocationsTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Tribe;
+
+/**
+ * Class Context_LocationsTest
+ *
+ * @since   TBD
+ *
+ * @package Tribe
+ */
+class Context_LocationsTest extends \Codeception\TestCase\WPTestCase {
+	/**
+	 * @test
+	 */
+	public function is_main_query_should_not_error_on_wp_query_null() {
```

Comment:
Do you think we need to do a `::tearDown()` where we `unset()` `$GLOBALS['wp_query']` and `$GLOBALS['wp_the_query']` if they are set?